### PR TITLE
feat: adds light themed header option

### PR DESCRIPTION
**Pull request by Golodhros**
### Summary of Changes
Adds an option to make the navigation into a light theme
Tweaks styles, spacing and header elevation

### Screenshots
Dark header with no logo
header-now-dark-no-logo

Light header with no logo
header-now-light-no-logo

Dark header with logo
header-now-with-logo

### Documentation
Updated config docs
Added Storybook story:

AMS

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

File: frontend/amundsen_application/static/js/features/NavBar/navBar.story.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  '/search': [
+    {
+      isFeatureTour: true,
+      isShownOnFirstVisit: true,
+      isShownProgrammatically: false,
+      steps: [
+        {
+          target: '.nav-bar-left a',
+          title: 'Title about the logo',
+          content: 'Content about the step pointing to the log',
```
Comment (kristenarmes) [inline frontend/amundsen_application/static/js/features/NavBar/navBar.story.tsx:78]: log -> logo?

File: frontend/amundsen_application/static/js/features/NavBar/navBar.story.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        },
+        {
+          target: '.search-filter-section-header #column',
+          title: 'Filters',
+          content: 'Info about Filters',
+        },
+        {
+          target: '#search-input',
+          title: 'Search ranking',
+          content: 'Search raking information',
```
Comment (kristenarmes) [inline frontend/amundsen_application/static/js/features/NavBar/navBar.story.tsx:88]:
```suggestion
          content: 'Search ranking information',
```

**Review by kristenarmes — APPROVED** [review]
LGTM

File: frontend/amundsen_application/static/js/features/NavBar/navBar.story.tsx (hunk 1 above)
Comment (Golodhros) [reply frontend/amundsen_application/static/js/features/NavBar/navBar.story.tsx:78]: good catch!

**Pull request merged**